x/leveragelp/keeper/position.go

```diff
@@ -114,60 +67,6 @@ func (k Keeper) DestroyPosition(ctx sdk.Context, positionAddress string, id uint
 	return nil
 }
 
-// Set sorted liquidation
-func (k Keeper) SetSortedLiquidation(ctx sdk.Context, address string, old sdk.Int, new sdk.Int) {
-	store := ctx.KVStore(k.storeKey)
-	if store.Has([]byte(address)) {
-		key := store.Get([]byte(address))
-		if !store.Has(key) {
-			return
-		}
-		res := store.Get(key)
-		var position types.Position
-		k.cdc.MustUnmarshal(res, &position)
-		// Make sure liability changes are handled properly here, this should always be updated whenever liability is changed
-		liquidationKey := types.GetLiquidationSortKey(position.AmmPoolId, position.LeveragedLpAmount, old, position.Id)
-		if len(liquidationKey) > 0 && store.Has(liquidationKey) {
-			store.Delete(liquidationKey)
-		}
-
-		// Add position sort keys
-		addrId := types.AddressId{
-			Id:      position.Id,
-			Address: position.Address,
-		}
-		bz := k.cdc.MustMarshal(&addrId)
-		liquidationKey = types.GetLiquidationSortKey(position.AmmPoolId, position.LeveragedLpAmount, new, position.Id)
-		if len(liquidationKey) > 0 {
-			store.Set(liquidationKey, bz)
```

Comment:
If we have removed this, let's remove it's Prefix key it as well.

Reply:
it is being used in migration, will be removed after upgrade, I have added comments too